testing/sdk_cmd.py

```diff
@@ -397,6 +412,8 @@ def _scp(
             # -oConnectTimeout=#: Limit the duration for the connection to be created.
             #                     We also configure a timeout for the command itself to run once connected, see below.
             "-oConnectTimeout={}".format(timeout_seconds),
+            # -i <identity_file>: The identity file to use for login
+            "-i {}".format(SSH_KEY_FILE) if SSH_KEY_FILE else "",
```

Comment:
Nit: There seems to be some overlap between these args and those used in `_ssh`. Could we have a function that returns them.

Reply:
I would prefer to do this in a separate cleanup commit / PR, but I can do it here if you think it's necessary.

Reply:
A separate one or a "note-to-self" is fine.

Reply:
After looking more closely at this, there are only 2 flags that scp and ssh share, and I'm not sure how I'd actually split them out to have a helper function without it convoluting the code. Sometimes its preferable to duplicate if it means maintenance and verification of correctness are more isolated.

Reply:
Yes. There isn't too much overlap, so the advantage is minimal.